File: Mayúscula y minúscula de un caracter/Python/pasa_a_minuscula.py
#Creado por Andrés Melgar
#para iterando++ http://iterando.online/
#entrada: http://iterando.online/ejercicios-resueltos/pasar-a-mayuscula-minuscula-un-caracter-en-python/
c=input("Ingrese caracter en mayúscula: ")
if c>='A' and c<='Z':
    c=chr(ord(c)-ord('A')+ord('a'))
print("En minúscula es: {}".format(c))

File: Mayúscula y minúscula de un caracter/Java/Pasa_a_Minuscula.java
/* Creado por Andrés Melgar
 * para iterando++ http://iterando.online/
 * entrada: 
 */
package pasa_a_minuscula;

import java.util.Scanner;

public class Pasa_a_Minuscula {

    public static void main(String[] args) {
        char c;
        Scanner reader = new Scanner(System.in);
        System.out.printf("Ingrese caracter en mayúscula: ");
        c = reader.next().charAt(0);

        if (c >= 'A' && c <= 'Z') {
            c = (char)(c - (int) 'A' + (int) 'a');
        }
        System.out.printf("En minúscula es: %c%n", c);
    }
    
}
